bindings/python/tox.ini

```diff
@@ -15,4 +15,14 @@ passenv = *
 deps = -r requirements-dev.txt
 commands =
     pip install .
-    pytest
\ No newline at end of file
+    pytest
+
+[testenv:lint]
+description = Run linter {basepython}
+deps = autopep8
+commands =
+    autopep8 --exclude .venv  --diff --recursive --aggressive --exit-code .
```

Comment:
Is this checking for format or applying it ?

Reply:
Only checking. Do we want to modify?

Reply:
No no, checking is fine